felix/daemon/daemon.go

```diff
@@ -485,7 +487,7 @@ configRetry:
 	// which will feed the calculation graph with updates, bringing Felix into sync.
 	var syncer Startable
 	var typhaConnection *syncclient.SyncerClient
-	syncerToValidator := calc.NewSyncerCallbacksDecoupler()
```

Comment:
Is NewSyncerCallbacksDecoupler not needed now?  If so, does that mean there's code that can be deleted?

Reply:
SyncerCallbacksDecoupler is still used elsewhere; downstream of the new dedupe buffer (so back pressure there will push back on the dedupe buffer anyway, indirectly).  I think we only want one dedupe buffer at the updates source so that we dedupe as early as possible.  If we did it later, I think we'd get undesirable interactions between them.